operate/importer-8_7/src/main/java/io/camunda/operate/zeebeimport/v8_7/processors/processors/VariableZeebeRecordProcessor.java

```diff
@@ -53,15 +53,15 @@ public void processVariableRecords(
       final var scopedVariables = variableRecords.getValue();
 
       for (final var scopedVariable : scopedVariables) {
+        if (!shouldProcessVariableRecord(scopedVariable)) {
+          continue;
```

Comment:
But we will then also skip update of `processDefinitionKey` and `bpmnProcessId` fields, right?
https://github.com/camunda/camunda/pull/29639/files#diff-c84c359a0aa539ce1522a9e4ec7a7633ecea43e94c09d6075febe516c6bbacb1R78-R79

May be we should get rid of this grouping fully and just process all the records that come? This grouping was done at times for performance reasons, but I think we already lost the essential part of it when we decided [whether we should index or upsert](http://github.com/camunda/camunda/commit/5f772471570e92ec609d14fee9c018a780cf7729#diff-bc2c80941beb6328c56cec34ef9697549fea3fe0edcb0d6bdc9f92d03f70574bR84-R88).